flask-backend/api/routes/case.py

```diff
@@ -117,6 +118,53 @@ def openFile(case_name, folder_name, file_name):
     return File
 
 
+@case.route('/filter', methods=['POST'])
+def filter():
+    '''
+        Route to filter the cases in range of date.
+        If starting date will not be provided it will 
+        return cases till the end date.
+        If both not provided all cases will be shown.
+        If starting date is provided and no end date then 
+        cases from starting date till present date will be shown.
+    '''
+    try:
+        req = request.get_json()
+
+        from_date = str(req['starting date'])
+
+        to_date = str(req['end date'])
+
```

Comment:
@swapnalshahil you can add an additional check for equal vales of to_date from from_date :)

Reply:
Hey for getting cases of only a particular date we have to write the same date on "starting date" as well as on the "end date". I thought about it earlier but I will use this filter in the analytics section to filter cases within range.

Reply:
Cool,  got it 👍🏼